doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -105,6 +105,10 @@ nRF9160
 
     * Added API to enable modem domain events.
 
+  * :ref:`https_client` sample:
+
+    * Added a possibility to use TF-M and Zephyr Mbed TLS instead of using the offloaded TLS stack in modem.
+
```

Comment:
If I remember correctly we did not track changes to samples in the changelog?

Reply:
Why not? It's a significant change to the sample, and we do have changelog entries for samples in the past release notes.

Reply:
Ok, I remember wrongly then.